skbeam/core/correlation.py

```diff
@@ -1086,21 +1075,20 @@ def _cross_corr(img1, img2=None, axes=None):
             to the right of img1 will lead to a shift of the point of
             highest correlation to the right.
             Default is set to None
-
-        axes : 1 or 2-tuple
-            the axis or axes to perform the cross correlation on
     '''
-    if axes is None:
-        if img1.ndim == 2:
-            axes = -2, -1
-        else:
-            axes = -1,
+    ndim = img1.ndim
 
     if img2 is None:
         img2 = img1
 
-    imgc = fftshift(ifft2(fft2(img1, axes=axes) *
-                    np.conj(fft2(img2, axes=axes)), axes=axes).real)
+    if img1.shape != img2.shape:
+        errorstr = "Image shapes don't match. "
+        errorstr += "(img1 : {},{}; img2 : {},{})"\
+            .format(*img1.shape, *img2.shape)
+        raise ValueError()
+
+    reverse_index = (*((slice(None, None, -1),)*ndim),)
```

Comment:
I guess this will fail in python2 which we currently are still supporting. Maybe try this
```
reverse_index = [slice(None, None, -1) for i in range(ndim)]
```